recipes/trompeloeil/all/conanfile.py

```diff
@@ -0,0 +1,32 @@
+from conans import ConanFile, tools
+import os
+import glob
+
+
+class TrompeloelConan(ConanFile):
+    name = "trompeloeil"
+    description = "Header only C++14 mocking framework"
+    topics = ("conan", "trompeloeil", "header-only", "mocking")
+    homepage = "https://github.com/rollbear/trompeloeil.git"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Boost Software License - Version 1.0 - August 17th, 2003"
```

Comment:
```suggestion
    license = "BSL-1.0"
```

Use SPDX license identifier https://spdx.org/licenses/